model/AI/Classes/AIPackage.md

```diff
@@ -0,0 +1,107 @@
+-------
+SPDX-License-Identifier: Community-Spec-1.0
+
+# AIPackage
+
+## Description
+
+Metadata information that can be added to a package to describe an AI application or trained AI model. 
+
+## Metadata
+
+- SubclassOf: Software/Package
+						Defaults are: 0..*
```

Comment:
What is that? this will break parsing.

Reply:
Thanks, added metadata per your suggestion